assignment4/ques3.js

```diff
@@ -0,0 +1,21 @@
+// Write a program to give count of variables in a string ("abcabcdabbcc" => "a3b4c4d1")
+let inputString = "abcabcdabbcc";
+
+let countMap = new Map();
+
+for (let i = 0; i < inputString.length - 1; i++) {
+    if (countMap.has(inputString[i])) {
+        countMap.set(inputString[i], countMap.get(inputString[i]) + 1)
+
+    }
+    else {
+        countMap.set(inputString[i], 1);
+    }
+}
+
+let result = "";
+countMap.forEach(function (val, key) {
```

Comment:
you can use the systax like 
countMap.forEach((val, key) => {
    result += key + val;

})